src/main/java/core/basesyntax/PrintInfo.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface PrintInfo {
+    void printInfo();

```

Comment:
```suggestion
public interface Drawable {
    void draw();

```
![image](https://github.com/mate-academy/jv-oop-advanced/assets/20326858/6db2cc36-3305-4532-900d-95c40cc52a77)
В коді завжди треба використовувати мову і термінологію замовника. Уявляєм. Ти приходиш працювати на проект. Тобі замовник каже We want to draw our Figures in new way! І описує вимоги. Добре, ти шукаєш в коді (якого в тебе гігабайти) Figure і Draw. Figure є, а Draw нема. Є printInfo. Тоді ти починаєш ломати голову, чи Draw і printInfo це теж саме, чи ні? Не слід створювати такі ситуації. Кожне слово і кожна буква в іменах та назвах є дуже важливими